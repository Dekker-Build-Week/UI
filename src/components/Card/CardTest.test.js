import React from 'react';

import Card from './Card';
import CardAvatar from './CardAvatar';
import CardBody from './CardBody';
import CardFooter from './CardFooter';
import CardHeader from './CardHeader';
import CardIcon from './CardIcon';
import renderer from 'react-test-renderer';

describe('Card test suite', () => {
    it('Renders the Card correctly', () => {
        const tree = renderer
            .create(<Card></Card>)
            .toJSON();

        expect(tree).toMatchSnapshot();
    }),
    it('Renders the CardAvatar correctly', () => {
        const tree = renderer
            .create(<CardAvatar><p>Hey</p></CardAvatar>)
            .toJSON();
    
        expect(tree).toMatchSnapshot();
    }),
    it('Renders the CardBody correctly', () => {
        const tree = renderer
            .create(<CardBody></CardBody>)
            .toJSON();
    
        expect(tree).toMatchSnapshot();
    }),
    it('Renders the CardFooter correctly', () => {
        const tree = renderer
            .create(<CardFooter></CardFooter>)
            .toJSON();
    
        expect(tree).toMatchSnapshot();
    }),
    it('Renders the CardHeader correctly', () => {
        const tree = renderer
            .create(<CardHeader></CardHeader>)
            .toJSON();
    
        expect(tree).toMatchSnapshot();
    }),
    it('Renders the CardIcon correctly', () => {
        const tree = renderer
            .create(<CardIcon></CardIcon>)
            .toJSON();
    
        expect(tree).toMatchSnapshot();
    })
});