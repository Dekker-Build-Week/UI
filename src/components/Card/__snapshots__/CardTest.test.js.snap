// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card test suite Renders the Card correctly 1`] = `
<div
  className="makeStyles-card-1"
/>
`;

exports[`Card test suite Renders the CardAvatar correctly 1`] = `
<div
  className="makeStyles-cardAvatar-5"
>
  <p>
    Hey
  </p>
</div>
`;

exports[`Card test suite Renders the CardBody correctly 1`] = `
<div
  className="makeStyles-cardBody-8"
/>
`;

exports[`Card test suite Renders the CardFooter correctly 1`] = `
<div
  className="makeStyles-cardFooter-11"
/>
`;

exports[`Card test suite Renders the CardHeader correctly 1`] = `
<div
  className="makeStyles-cardHeader-16"
/>
`;

exports[`Card test suite Renders the CardIcon correctly 1`] = `
<div
  className="makeStyles-cardIcon-26"
/>
`;

exports[`renders correctly 1`] = `
<div
  className="makeStyles-card-1"
/>
`;
